tests/create_property_group_test.py

```diff
@@ -55,6 +58,9 @@ def test_create_property_group():
         assert (
             workspace.find_data(single_data_group.properties[0]).name == f"Period{1}"
         ), "Failed at creating a property group by data name"
+
+        # Remove on props from the list
+        curve.remove_data_from_group(props[-1], "myGroup")
```

Comment:
**suggestion:** add a comment about what this line is supped to test? (I do not get it)

Or maybe have a dedicated test function just for instead of being buried in a larger test?

Reply:
New test created isolating the modification of prop_groups